README.md

```diff
@@ -1,2 +1,346 @@
+
+## The basics 
+<ul> 
+    <li> The file you create should end with .html </li>
+    <li> To play(TO DO: find a more suitable verb) the file you just open it in a web browser (Internet Explorer, Firefox, Chrome, Safari, Opera) </li>
+    <li> HTML doesn't include styling of the page(that's what CSS is for) </li>
+</ul>
+
+Okay, so we write html in a text editor. Then how does our PC know it's a HTML file and not a poem or essay we've written? 
+Easy! HTML has it's own syntax( it's very close to the English language, don't worry) and it's own extension - **_.html_**.
+
+## HTML head and body
+
+Every html starts with the tag 
+``` html
+<!DOCTYPE html>
+``` 
+This way the browser recognizes that this is a valid .html file.
+You're probably wondering what a tag is.
+<blockquote>A tag is 
+
+``` html
+<something here inside>
+```
+
+it consists of an opening tag and usually (not every tag has one) a closing tag:
+
+``` html 
+</something that was in the opening tag>
+```
+
+Eveything that's between the opening and the closing tag is affected by the action this tag does. The 
+
+``` html 
+<!DOCTYPE html>
+```
+
+doesn't have a closing tag and neither does 
+
+``` html 
+<br>
+```
+Br is the tag for new line.  </blockquote>
+
+After that our html has 
+
+``` html 
+<html> tag
+```
+
+The html tag has a closing tag:
+
+``` html 
+</html>
+```
+
+In between the html tags, our code looks just like a human - it starts with
+
+``` html 
+<head> some code here </head>
+```
+
+and after the head we have
+
+``` html 
+<body> some code here </body>
+```
+
+ Between the head tags we place the head elements.
+
+### The following elements go inside the head element:
+<blockquote>
+title = what is shown at the tab in your browser <br>
+style = here we put rules for styling (in another language called CSS) <br>
+link = links <br>
+meta =meta information like encoding <br>
+script = actions for you page <br>
+</blockquote>
+
+Most of the things you will want to add (like sentences, pictures, etc) 
+will be added in the body tag. Now lets learn more about them.
+
+## Paragraphs
+The easiest thing you can do in HTML is write a sentence. 
+To do this you need to know one of the very basic and easy to use tags - the
+
+``` html 
+<p>
+```
+
+To write a sentence(or paragraph, word, number, etc), we simply put what we want to be shown
+on the screen between an opening and a closing tag.
+
+``` html 
+<p> Your text goes here </p>
+```
+ The paragraph tag should be used in the body. 
+
+<b> Extra knowledge: </b> HTML doesn't add new lines
+when you press Enter in your editor. To add a new line, we use the 
+
+``` html 
+<br>
+```
+
+tag. **N.B!**The br tag has only an opening tag!! 
```

Comment:
It's actually known as a 'self-closing' tag to be strictly correct about it 😉 

I think we're also missing the closing `**` here to finish the bold portion of this section.